languages/en.ini

```diff
@@ -433,8 +437,21 @@ fav_list_delete_fail = "Sorry, an error has occurred. Your list was not deleted.
 Favorites = "Saved Items"
 Fee = "Fee"
 Feedback = "Feedback"
+Feedback Management = "Feedback Management"
+feedback_delete_failure = "Feedback deletion failed"
+feedback_delete_filter = "You are using following filter - Form name: %formname%, Site URL: %siteurl%, Status: %status%"
+feedback_delete_success = "Feedback deleted: %count%"
```

Comment:
I wonder if `"%count% feedback responses deleted."` would sound more natural. Also note that we normally use double-percents for placeholders (e.g. `%%count%%`) but your code seems to be using single percents. I think it's worth adjusting that for consistency.

Reply:
I changed the translation and also token identifiers for consistency.